Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -28,7 +28,7 @@ public final class SecureEnclaveValet: NSObject {
     // MARK: Public Class Methods
 
     /// - Parameters:
-    ///   - identifier: A non-empty string that uniquely identifies a SecureEnclaveValet.
+    ///   - identifier: A non-empty string that must correspond with the value for keychain-access-groups in your Entitlements file.
```

Comment:
This isn't a shared access group Valet:
```suggestion
    ///   - identifier: A non-empty string that uniquely identifies a SecureEnclaveValet.
```